cf-java-monitoring-custom-metrics-clients/cf-custom-metrics-clients-java/src/main/java/com/sap/cloud/cf/monitoring/java/converter/CounterConverter.java

```diff
@@ -0,0 +1,18 @@
+package com.sap.cloud.cf.monitoring.java.converter;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map.Entry;
+
+import com.codahale.metrics.Counter;
+import com.sap.cloud.cf.monitoring.client.model.Metric;
+
+public final class CounterConverter extends MetricConverter<Counter> {
+
+    @Override
+    List<Metric> convertMetricEntry(Entry<String, Counter> metricEntry, long timestamp) {
+        Counter counter = metricEntry.getValue();
+        return Arrays.asList(buildCustomMetric(metricEntry.getKey() + ".count", counter.getCount(), MetricType.COUNTER,

```

Comment:
It is just in-line array. Oneday if we need to add more quantiles could be refactored

Reply:
What bothers me is the different behaviour of the List types, that are generated. List generated by `Arrays.asList(...)` cannot be added to, but their elements can be replaced. Since Java does not have an interface for a read-only list (at least in Version 8), returning this kind of List may lead to unwanted behaviour in downstream code.

Reply:
I will keep the current implementation, if you don't have any other concerns